src/store/factory/client.js

```diff
@@ -177,7 +177,7 @@ function createSolanaClient (network, mnemonic, derivationPath) {
 function createRskClient (asset, network, mnemonic, accountType, derivationPath) {
   const isTestnet = network === 'testnet'
   const rskNetwork = ChainNetworks.rsk[network]
-  const rpcApi = isTestnet ? 'https://public-node.testnet.rsk.co' : 'https://public-node.rsk.co'
+  const rpcApi = isTestnet ? 'https://testnet.sovryn.app/rpc' : 'https://mainnet.sovryn.app/rpc'
```

Comment:
There are also other occurances of this in the wallet, please update / consolidate

Reply:
got it, let me check right now
Thanks!

Reply:
@monokh done, it's fixed now, added an env var to set mainnet and testnet endpoints then if we need to update in the future only the env var should be updated